### PR TITLE
Add note on upgrade paths for NOM

### PR DESCRIPTION
**Pull request by SaschaPeukert**
----

If you open a PR that needs to go into a current version, you need to *cherry-pick your commit from dev over to the current version branch*. Only then will the proper builds that generate html/pdf be run. But beware: Docs will be generated but not published automatically!

- [x] N/A - or - I have added the appropriate "cherry-pick-to" labels to this PR so I don't forget to do this later!

### CODE REVIEW

File: modules/ROOT/pages/update/index.adoc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 = Updating to a newer version
 :description: This section describes the upgrade process from an earlier version of NOM.
 
-It is recommended to always keep NOM up-to-date with the latest available version.
+It is recommended to always keep NOM up-to-date with the latest available version. You should be able to update from any older NOM version to any newer version. A version downgrade is not possible. The natural order of NOM versions follows our
```
Comment (SeBBBe) [inline modules/ROOT/pages/update/index.adoc]:
```suggestion
It is recommended to always keep NOM up-to-date with the latest available version. You can update from any older NOM version to any newer version. A version downgrade is not possible. The natural order of NOM versions follows our
```

Comment (SeBBBe) [reply modules/ROOT/pages/update/index.adoc]: Let's project some more confidence here! 😄

**Review by eve-bright — APPROVED** [review]

File: modules/ROOT/pages/appendix/version-compatibility.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 = Version compatibility
 :description: This section provides a version compatibility matrix for Neo4j Ops Manager.
 
-== Versioning scheme
+== Versioning scheme [[versioning_scheme]]
```
Comment (AlexicaWright) [inline modules/ROOT/pages/appendix/version-compatibility.adoc]:
```suggestion
== Versioning scheme 
```

File: modules/ROOT/pages/appendix/version-compatibility.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 = Version compatibility
 :description: This section provides a version compatibility matrix for Neo4j Ops Manager.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/appendix/version-compatibility.adoc:3]:
```suggestion

[[versioning_scheme]]
```

File: modules/ROOT/pages/update/index.adoc [hunk 4]
```diff
@@ -1,7 +1,8 @@
 = Updating to a newer version
 :description: This section describes the upgrade process from an earlier version of NOM.
 
-It is recommended to always keep NOM up-to-date with the latest available version.
+It is recommended to always keep NOM up-to-date with the latest available version. You can update from any older NOM version to any newer version. A version downgrade is not possible. The natural order of NOM versions follows our
```
Comment (AlexicaWright) [inline modules/ROOT/pages/update/index.adoc]:
```suggestion
It is recommended to always keep NOM up-to-date with the latest available version. 
You can update from any older NOM version to any newer version. 
A version downgrade is not possible. 
The natural order of NOM versions follows the *xref:../appendix/version-compatibility.adoc#versioning_scheme[semantic versioning scheme]*.
```

File: modules/ROOT/pages/update/index.adoc [hunk 5]
```diff
@@ -1,7 +1,8 @@
...
+*xref:../appendix/version-compatibility.adoc#versioning_scheme[semantic versioning scheme]*.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/update/index.adoc]:
```suggestion

```

**Review by AlexicaWright — COMMENTED** [review]
One sentence per line and some suggestions.

**Review by SeBBBe — APPROVED** [review]

**Pull request merged**